extensions/msal-node-extensions/package.json

```
@@ -38,7 +38,7 @@
   "module": "dist/test.esm.js",
   "dependencies": {
     "@azure/msal-common": "^5.0.0",
-    "bindings": "^1.5.0",
+    "bindings": "git://github.com/samuelkubai/node-bindings.git#3a5a099ed0178b65079ce18e11fccc7130e0af1e",
```

```suggestion
    "bindings": "git://github.com/samuelkubai/node-bindings.git",
```

Do we need to reference the exact commit? (a tag might be better)

---

Good point, let me provide a tag instead.

---

I have updated the `package.json` file to use a tag instead of a commit hash.